cmd/swctl/options.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"os"
+	"strings"
+
+	"github.com/gookit/color"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+var (
+	glob GlobalOptions
+)
+
+type GlobalOptions struct {
+	Debug    bool
+	LogLevel string
+	Color    string
+	// TODO: support config file
+	// Config string
+}
+
+func InitGlobalOptions(cli Cli, opts *GlobalOptions) {
+	// color mode
+	if opts.Color == "" && os.Getenv("NO_COLOR") != "" {
+		// https://no-color.org/
+		opts.Color = "never"
+	}
+	switch strings.ToLower(opts.Color) {
+	case "auto", "":
+		if !cli.Out().IsTerminal() {
+			color.Disable()
+		}
+	case "on", "enabled", "always", "1", "true":
+		color.Enable = true
+	case "off", "disabled", "never", "0", "false":
+		color.Disable()
+	default:
+		logrus.Fatalf("invalid color mode: %q", opts.Color)
+	}
+
+	// debug mode
+	if os.Getenv("SWCTL_DEBUG") != "" {
+		opts.Debug = true
+	}
+
+	// log level
+	if loglvl := os.Getenv("SWCTL_LOGLEVEL"); loglvl != "" {
```

Comment:
The env variable names are scattered across this package. It would be better to centralize them with constant block, or at least make a constant block in each file where they are (this and cli.go).